patches/node/fix_if_not_already_loaded_load_advapi32.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Felix Rieseberg <felix@felixrieseberg.com>
+Date: Tue, 9 Jun 2020 09:10:17 -0700
+Subject: fix: If not already loaded, load advapi32
+
+As of Electron 9, we have a dependency on advapi32.dll. That dll contains
+everything Windows would consider "advanced", like the registry, Windows
+services, user account management, and some other stuff. It's fair that
+Node expects the module to be loaded.
+
+Chrome explicitly delay-loads it, so on some machines, we've seen Electron
+crash hard because the "gimme advapi32" code is running before the dll
+was loaded. It'll be loaded either way, this patch ensures that we do so
+when we haven't already.
```

Comment:
Can we upstream this?

Reply:
Unlikely, we could try.  Upstream doesn't need to handle the fact this module is lazy loaded :shrug: